docs/_templates/layout.html

```diff
@@ -142,10 +142,10 @@
       <h1>Settings</h1>
 
       <div class='setting'>
-        <h3>Use a sans-serif font:
+        <h3>Use a serif font:
           <label class="toggle"
-            title="Use a sans serif font? Your system font will be used, falling back to `sans-serif`.">
-            <input type="checkbox" name="useSansFont" onclick="updateSetting(this);">
+            title="Use a serif font? Your system font will be used, falling back to `serif`.">
```

Comment:
These shouldn't be here.
```suggestion
            title="Use a serif font? Your system font will be used, falling back to serif.">
```